packages/core/src/main/notification/NotificationService.ts

```diff
@@ -88,11 +87,14 @@ export class NotificationService extends EventEmitter {
     return TOPIC;
   }
 
-  constructor(apiClient: APIClient, cryptographyService: CryptographyService) {
+  constructor(
+    apiClient: APIClient,
+    cryptographyService: CryptographyService,
+    private readonly storeEngine: CRUDEngine,
```

Comment:
Why did you move this here? IMO outside of the constructor was the better place.